docs/sources/operations/request-validation-rate-limits.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Request Validation & Rate-Limit Errors
+weight: 30
+---
+
+# Request Validation & Rate-Limit Errors
+
+Loki will reject requests if they exceed a usage threshold (rate-limit error) or if they are invalid (validation error).
+
+All occurrences of these errors can be observed using the `loki_discarded_samples_total` and `loki_discarded_bytes_total` metrics. The sections below describe the various possible reasons specified in the `reason` label of these metrics.
+
+It is recommended that Loki operators set up alerts or dashboards with these metrics to detect when rate-limits or validation errors occur. 
+
+
+### Terminology
+
+- **sample**: a log line
+- **series**/**stream**: samples with a unique combination of labels
+- **active stream**: streams that are present in the ingesters - these have recently received log lines within the `chunk_idle_period` period (default: 30m)
+
+## Rate-Limit Errors
+
+Rate-limits are enforced when Loki cannot handle more requests from a tenant.
+
+### `rate_limited`
+
+This rate-limit is enforced when a tenant has exceeded their configured log ingestion rate-limit.
```

Comment:
It would fire for either the baseline rate-limit or the burst.

> Is there a way to differentiate between the two?

I don't believe so. The implementation is handled in https://pkg.go.dev/golang.org/x/time/rate#Limiter.AllowN, and it doesn't specify if the burst or baseline is violated AFAICS from a quick scan.

Reply:
Understood. Funny enough I see @owen-d elsewhere is suggesting we get rid of the 'burst' limit. 
I guess let's just leave as is for now and its sorta up to the user to have to figure out which limit to raise (or if they raise both). 
In practice, what do we do when a customer hits this limit? Do we raise both? or just the `ingestion_rate_mb`? Does the burst size limit increase for example as I go from medium to large user?